audit/langcard.asm

```diff
@@ -49,6 +49,24 @@ LANGCARDTESTS_NO_CHECK:
 	+prerred
 	sec
 	rts
++	lda $C08B		; Read and write bank 1 with single access (only one needed if banked in already)
```

Comment:
FYI, I had a lot of explicit tests here, and tried to move anything but sanity checks down into the [data-driven tests](https://github.com/peterferrie/a2audit/blob/f84b9f5e8e967093bd443047b7b4627d55454220/audit/langcard.asm#L297) below. Would these be better off there?